R/namespace.R

```diff
@@ -268,9 +268,13 @@ roxy_tag_ns.roxy_tag_importFrom <- function(x, block, env) {
   pkg <- x$val[1L]
   if (requireNamespace(pkg, quietly = TRUE)) {
     importing <- x$val[-1L]
-    unknown_idx <- !importing %in% getNamespaceExports(pkg)
+    # be sure to match '%>%', `%>%`, "%>%" all to %>% given by getNamespaceExports, #1570
+    unknown_idx <- !normalize_quotes(importing) %in% getNamespaceExports(pkg)
     if (any(unknown_idx)) {
-      warn_roxy_tag(x, "Excluding unknown {cli::qty(sum(unknown_idx))} export{?s} in from {.package {pkg}}: {.code {importing[unknown_idx]}}")
+      warn_roxy_tag(x, "Excluding unknown {cli::qty(sum(unknown_idx))} export{?s} from {.package {pkg}}: {.code {importing[unknown_idx]}}")
+      if (all(unknown_idx)) {
+        return(NULL)
+      }
       x$val <- c(pkg, importing[!unknown_idx])
     }
   }
```

Comment:
One more thought: how do the different types of quoting affect the `NAMESPACE`? Are they normalised elsewhere, or is it ok to use any of the forms without quoting?

Reply:
Do you mean from roxygen2 perspective or base R?

Answering for the latter: NAMESPACE needs to parse as R, so `importFrom(magrittr, %>%)` won't work. 

So for roxygen2, my understanding is `@importFrom magrittr %>%` will add quotes for convenience (that's the helper `auto_quotes`), but quoted forms are dropped in as-is.